README.md

```diff
@@ -4,14 +4,15 @@ This repository accompanies Plaid's [**quickstart guide**][quickstart].
 
 Here you'll find full example integration apps using our [**client libraries**][libraries].
 
-This Quickstart is designed to show as many products and configurations as possible, including all five officially supported backends and multiple Plaid APIs, against a React frontend. 
+This Quickstart is designed to show as many products and configurations as possible, including all five officially supported client libraries and multiple Plaid APIs, against a React frontend. 
```

Comment:
FWIW, I was never really a huge fan of the phrase "client library" because that sounds like something you would use on the actual client. Should we think of a better name to start using? "Server libraries"?